src/content/1.7/development/webservice/tutorials/2-testing-access.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 2 - Testing access to the Webservice
+menuTitle: 2 - Testing access
+weight: 2
+---
+
+# Testing access to the Webservice
+
+## Accessing /api/
+
+Now that your access key is generated you can test your store's webservice, its endpoint is located in the `/api/` folder at the root of your installation of Prestashop.
+The quickest way to test your API is to use your browser:
+
+* If PrestaShop is installed at the root of your server, you can access the API here: http://example.com/api/
+* If PrestaShop is installed in a subfolder of your server, you can access the API here: http://example.com/prestashop/api/
+
+The shop should prompt you for a username and a password to enter. The username is the authentication key you created and **there is no password to enter**. If no permission has been set for the key, than the browser will keep asking you to enter the key indefinitely.
```

Comment:
> If no permission has been set for the key, than the browser will keep asking you to enter the key indefinitely.

It looks like a bug. Should return a 403 forbidden :thinking:

Reply:
That's what happens if you hit Cancel, the server returns a 401 with an xml message